src/encrypt.js

```diff
@@ -161,7 +166,7 @@ function createInfo(type, context) {
  * @param  {Buffer} ikm    Input keying material
  * @param  {Buffer} info   Application-specfic context
  * @param  {Number} length The length (in bytes) of the required output key
- * @return {Buffer}
+ * @return {Buffer} hkdf
```

Comment:
The value returned is actually a *key*. HKDF is the name of the function itself.